common/src/main/java/com/wynntils/features/user/NpcDialogAutoProgressFeature.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.user;
+
+import com.google.common.collect.ImmutableList;
+import com.wynntils.core.chat.ChatModel;
+import com.wynntils.core.config.Config;
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.core.features.properties.RegisterKeyBind;
+import com.wynntils.core.features.properties.StartDisabled;
+import com.wynntils.core.keybinds.KeyBind;
+import com.wynntils.core.managers.Model;
+import com.wynntils.mc.utils.McUtils;
+import com.wynntils.wynn.event.NpcDialogEvent;
+import java.util.Optional;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+import net.minecraft.network.protocol.game.ServerboundPlayerCommandPacket;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import org.lwjgl.glfw.GLFW;
+
+@StartDisabled
+public class NpcDialogAutoProgressFeature extends UserFeature {
+    public static NpcDialogAutoProgressFeature INSTANCE;
+
+    @RegisterKeyBind
+    public final KeyBind cancelAutoProgressKeybind =
+            new KeyBind("Cancel Dialog Auto Progress", GLFW.GLFW_KEY_Y, false, this::cancelAutoProgress);
+
+    private final ScheduledExecutorService executor = Executors.newSingleThreadScheduledExecutor();
+
+    private final Runnable sendShiftRunnable = () -> {
+        McUtils.sendPacket(new ServerboundPlayerCommandPacket(
```

Comment:
I thought it was the action of sneaking that was causing the dialog to progress. Or is this Vanilla's weird way of handling sneak, even if it is remapped?

Reply:
Yes, this is very likely a source code thing, or perhaps the mapping is "wrong".